src/runtime/TemporalObject.cpp

```diff
@@ -273,7 +271,7 @@ TemporalObject::DateTime TemporalObject::parseValidIso8601String(ExecutionState&
         }
 
         // Calendar
-        if (isoString.rfind("[u-ca=") == 0) {
+        if (isoString.rfind("[u-ca=") != std::string::npos) {
```

Comment:
Can it be anywhere in the string?

Reply:
No. It need to be started at index. I will fix it.